include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
It makes sense though, and it fixes the problem. The reason is that StandardCLibrary is included in CppUTestConfig halfway. It must be included at *that* point. Having the CppUTestConfig before the include guard would prevent StandardCLibrary from being included, so it must be actually outside of the include guide.

That way the problem is served permanently.